### PR TITLE
Contact form to my Portfolio

### PR DESCRIPTION
**Pull request by pratap-panabaka**
Adding the "Contact Form" section to my Portfolio

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -195,28 +195,26 @@ <h3 class="connect">LET’S CONNECT</h3>
...
 		<footer>
 			<section class="contact-me">
-				<h1 class="contact-me-text">contact me</h1>
+				<h1 class="contact-me-text">Contact me</h1>
 				<p class="contact-me-brief">If you have an application you are intrested in developing, a feature that
 					you need built or a project that needs coding.
 					I'd love to help with it.
 				</p>
-				<form action="">
+				<form action="https://formspree.io/f/my-portfolio" method="post">
```
Comment (ShahierNashaat) [inline index.html]: - [ ] Kindly be noticed that you should make an account on formspree and get a valid form-id so the form can work.

File: index.html [hunk 2]
```diff
@@ -195,28 +195,26 @@ <h3 class="connect">LET’S CONNECT</h3>
...
 					<div class="input-field-box">
-						<input class="input-field" type="text" name="NAME" maxlength="30" value="Type your name">
-						<input class="input-field" type="email" name="EMAIL" value="username@domain.ext">
+						<input class="input-field" type="text" name="name" maxlength="30" value="Type your name">
```
Comment (ShahierNashaat) [inline index.html]: - [ ] Kindly be noticed that you should replace the `value` attribute with the `placeholder` attribute and also you have to add the `required` attribute.

File: index.html [hunk 3]
```diff
@@ -195,28 +195,26 @@ <h3 class="connect">LET’S CONNECT</h3>
...
+						<input class="input-field" type="email" name="email" value="username@domain.ext">
```
Comment (ShahierNashaat) [inline index.html]: - [ ] Kindly be noticed that you should replace the `value` attribute with the `placeholder` attribute and also you have to add the `required` attribute.

File: index.html [hunk 4]
```diff
@@ -195,28 +195,26 @@ <h3 class="connect">LET’S CONNECT</h3>
...
 						<textarea class="write-message" name="write message" placeholder="Write your message here"
```
Comment (ShahierNashaat) [inline index.html]: - [ ] Kindly remove the `rows` and the `cols` attributes from the textarea and add the `maxlength="500"` attribute also you have to add the `required` attribute.

File: index.html [hunk 5]
```diff
@@ -195,28 +195,26 @@ <h3 class="connect">LET’S CONNECT</h3>
...
-							maximumlength="100" rows="10" cols="30">
-						</textarea>
+							rows="5" cols="30"></textarea>
 					</div>
 					<div class="get-in-touch">
 						<button type="button">Get in Touch</button>
```
Comment (ShahierNashaat) [inline index.html]: - [ ] Kindly noticed that the submit button can be either an `input` element with the type `submit` or a button element with the type `submit` or with no type attribute.

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @PRATAP-KUMAR , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by algerina — APPROVED** [review]
Hi @PRATAP-KUMAR , 
Good job so far! 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- Your project looks good:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous TSEs
 
 ## Optional suggestions
 
 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**